### PR TITLE
Fix GitHub actions workflow for ruby and miniconda

### LINKED ISSUE #175: Failing tests are because of a change in Github actions
**Describe the bug**
Tests are failing because of a change in github actions:
https://github.blog/changelog/2020-10-01-github-actions-deprecating-set-env-and-add-path-commands/

### PR DESCRIPTION
**Pull request by SarahAlidoost**
# Pull Request checklist

We appreciate your time and effort to improve the tutorial. Please keep in mind that lesson maintainers are volunteers and it may be some time before they can respond to your contribution.

* * *

## Before you start

- [x] Read [CONTRIBUTING.md](https://github.com/ESMValGroup/tutorial/blob/master/CONTRIBUTING.md).
- [x] Create an [issue](https://github.com/ESMValGroup/tutorial/issues) to discuss your idea. This allows your contributions to be incorporated into the tutorial.

## Tasks

- [x] Give this pull request a descriptive title.
- [ ] ~~If you are contributing to existing lesson materials, please make sure the content conforms to the `Lesson development` section in [CONTRIBUTING.md](https://github.com/ESMValGroup/tutorial/blob/master/CONTRIBUTING.md) and does not contain any spelling or grammatical errors.~~
- [ ] ~~If you are making a new episode, please make sure the content conforms to the `Lesson organization` and `Lesson formatting` sections in [CONTRIBUTING.md](https://github.com/ESMValGroup/tutorial/blob/master/CONTRIBUTING.md) and does not contain any spelling or grammatical errors.~~
- [x] Preferably Codacy checks pass. Status can be seen below your pull request. If there is an error, click the link to find out why.
- [ ] ~~Preview changes on your machine before pushing them to GitHub by running `make serve`, alternatively `make docker-serve`. Please see the `Previewing your changes locally` section in [CONTRIBUTING.md]~~(https://github.com/ESMValGroup/tutorial/blob/master/CONTRIBUTING.md) for installation instructions.
- [ ] ~~All code instructions have been tested.~~

If you need help with any of the tasks above, please do not hesitate to ask by commenting in the issue or pull request.

* * *

Closes #175, #166

The changes in this PR are based on:

- using [ACTIONS_ALLOW_UNSECURE_COMMANDS](https://github.com/actions/toolkit/issues/641).
- using [conda-incubator](https://github.com/ESMValGroup/ESMValCore/pull/873).

### CODE REVIEW

**Review by stefsmeets — COMMENTED** [review]
I'm reading in various places that this work-around is no longer necessary with newer versions of the action. Can you try with: `ruby/setup-ruby@v1.64.1` (latest version as of today) and see if that works?

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -11,6 +11,8 @@ jobs:
     - uses: actions/checkout@v2
     - name: Setup Ruby
       uses: ruby/setup-ruby@v1.38.0
```
Comment (stefsmeets) [inline .github/workflows/ci.yml:13]:
```suggestion
      uses: ruby/setup-ruby@v1.64.1
```

**Comment by SarahAlidoost** [comment]
> I'm reading in various places that this work-around is no longer necessary with newer versions of the action. Can you try with: `ruby/setup-ruby@v1.64.1` (latest version as of today) and see if that works?

Thanks, I couldn't find a fix in ruby versions but I will upgrade it to the version suggested by you and remove  ACTIONS_ALLOW_UNSECURE_COMMANDS. Let's see if it works.

**Comment by SarahAlidoost** [comment]
> > I'm reading in various places that this work-around is no longer necessary with newer versions of the action. Can you try with: `ruby/setup-ruby@v1.64.1` (latest version as of today) and see if that works?
> 
> Thanks, I couldn't find a fix in ruby versions but I will upgrade it to the version suggested by you and remove ACTIONS_ALLOW_UNSECURE_COMMANDS. Let's see if it works.

It is fixed by the newer version, thanks.

**Review by stefsmeets — APPROVED** [review]
Great to see that it works!

**Pull request merged**